exec/java-exec/src/main/java/org/apache/drill/exec/work/WorkManager.java

```diff
@@ -295,7 +295,7 @@ public FragmentExecutor getFragmentRunner(final FragmentHandle handle) {
    */
   private class StatusThread extends Thread {
     public StatusThread() {
-      setDaemon(true);
+      setDaemon(false);
```

Comment:
Not sure why removing it will not ensure the same. I think it's redundant and like overriding a default already set in constructor with default value using setter method.